Project/Assets/ML-Agents/Examples/Crawler/Scripts/CrawlerAgent.cs

```diff
@@ -91,17 +91,9 @@ public void CollectObservationBodyPart(BodyPart bp, VectorSensor sensor)
         //GROUND CHECK
         sensor.AddObservation(bp.groundContact.touchingGround); // Is this bp touching the ground
 
-        //Get velocities in the context of our orientation cube's space
-        //Note: You can get these velocities in world space as well but it may not train as well.
-        sensor.AddObservation(orientationCube.transform.InverseTransformDirection(bp.rb.velocity));
-        sensor.AddObservation(orientationCube.transform.InverseTransformDirection(bp.rb.angularVelocity));
-
-        //Get position relative to hips in the context of our orientation cube's space
-        sensor.AddObservation(orientationCube.transform.InverseTransformDirection(bp.rb.position - body.position));
-
         if (bp.rb.transform != body)
         {
-            sensor.AddObservation(bp.rb.transform.localRotation);
```

Comment:
Corresponds to local space rotations

Reply:
are these decisions that could have been made with the help of the tool @andrewcoh was talking about for saliency?

Reply:
In this case, I removed everything from CollectObservations that had an equivalent in the sensor. 

In general, we could turn everything on in the sensor, run the tool to see what's important, and then disable unimportant stuff. However:
1) It could be tedious to attribute a particular index in the tool to a specific option in the sensor.
2) The tool might be too fine-grained. For example, if it said that the w component of one quaternion wasn't relevant, we couldn't just turn that one off. Or if local space velocities was only relevant for one body, we couldn't keep that one but disable the rest.